DCF_Demo.md

```diff
@@ -0,0 +1,513 @@
+# DCF Invoke Device Method Using Azure CLI
+
+## Open up an instance of Azure CLI 
+You can use the Azure Portal or an instance on Powershell/bash/other local environments
+
+## Commands to Run 
+
+<details>
+<summary>Click here for Portal/bash commands</summary>
+<br>
+
+Query for existing interfaces on the device 
+```
+az iot hub invoke-device-method -n [name-of-iothub] -d [name-of-device] --mn "ipc_query.1.query" --mp "{}"
+
+// expected outcome
+{
+  "payload": {
+    "continuation_token": 655615,
+    "result": [
+      "ipc_query.1",
+      "dm.1"
+    ]
+  },
+  "status": 200
+}
+
+```
+
+
+Install sprinker interface
+```
+az iot hub invoke-device-method -n [name-of-iothub] -d [name-of-device] --mn "dm.1.install" --mp "{\"package_name\":\"sprinkler_v1i1\"}" 
+
+// expected outcome
+{
+  "payload": {},
+  "status": 200
+}
+```
```

Comment:
Let's add a ipc_query here to show that the new interface sprinkler.1 is now part of the available interfaces.

Reply:
added in c5d48459b23391e198b444a6831f5917a546909c